src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2014,6 +2046,15 @@ void MarkMethodIfNotNull (MethodReference method)
 
 		protected virtual void MarkMethodBody (MethodBody body)
 		{
+			if (_context.IsOptimizationEnabled (CodeOptimizations.LazyBodyMarking)) {
+				if (!body.Method.IsStatic 
```

Comment:
I'd like more simple method with the logic like

`if (_context.IsOptimizationEnabled (CodeOptimizations.LazyBodyMarking) && IsUnreachableBody (method.Body)) {`